src/services/refactors/convertFunctionToEs6Class.ts

```diff
@@ -20,10 +22,28 @@ namespace ts.refactor {
             symbol = (symbol.valueDeclaration as VariableDeclaration).initializer.symbol;
         }
 
-        return symbol && symbol.flags & SymbolFlags.Function && symbol.members && symbol.members.size > 0;
+        if (symbol && symbol.flags & SymbolFlags.Function && symbol.members && symbol.members.size > 0) {

```

Comment:
It seems like it would be very helpful to parenthesize `symbol.flags & SymbolFlags.Function`.